webui/src/data/server.js

```diff
@@ -652,6 +652,9 @@ class ServerCache {
         if (!disciplines) {
             this.getters.disciplines.autofetch();
         }
+        if(disciplines && disciplines.filter((v) => { if(v.id === "-") { return v.id } }).length == 0 ){
```

Comment:
I think `(v) => { if(v.id === "-") { return v.id } }` can be rewritten as `v => v.id === "-"` (will return true if the id is '-', false otherwise).